### PR TITLE
https://github.com/AdguardTeam/AdguardBrowserExtension/issues/117

### PR DESCRIPTION
**Pull request by Mizzick**

### CODE REVIEW

**Comment by ameshkov** [comment]
Why so? There is much more simpler solution.
Just look at _getAllAdguardFilters function. There is a "sort" call, just change it a bit to place user filter to the end of the list.

**Comment by Mizzick** [comment]
>_getAllAdguardFilters
There is no user filter in this collection.

**Comment by ameshkov** [comment]
> There is no user filter in this collection.

Orly?

File: Extension/lib/filter/antibanner.js [hunk 1]
```diff
@@ -278,15 +278,25 @@ AntiBannerService.prototype = {
             // Creates request filter
             var requestFilter = new RequestFilter();
 
+            var userFilterRules = [];
```
Comment (ameshkov) [inline Extension/lib/filter/antibanner.js]: Add a comment to the github issue about this

**Comment by ameshkov** [comment]
What do I suggest:
1. Remove ```CollectionUtils.getRulesFromTextUnique``` call
2. Replace dirtyRules with dirtyRulesMap
3. Change ```getRequestFilter`` method:
* Create a supplement object there to ensure rules uniqueness
* Control the rules load order there

**Comment by ameshkov** [comment]
Fixed it myself.

**Pull request closed without merging**